ARSIS-Unity/Assets/Scripts/Components/Elements/NotificationWindowManager.cs

```diff
@@ -0,0 +1,89 @@
+using ARSIS.EventManager;
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class NotificationWindowManager : MonoBehaviour
+{
+    private NotificationDisplayManager _notificationDisplayManager;
+    public GameObject MiniNotifObj;
+    public GameObject miniParentObject;
+    void Start()
+    {
+        _notificationDisplayManager = FindObjectOfType<NotificationDisplayManager>();
+    }
+
+    void Update()
+    {
+        SetMenu();
+    }
+
+    void SetMenu()
+    {
```

Comment:
```suggestion
    {
        foreach (Transform child in mainParentObject.transform)
        {
            Destroy(child.gameObject);
        }
```
As a placeholder before deduplicating and storing the interaction status (has the user dismissed a notification). After running for a while this can have a performance impact if we dont clear the old notifications